src/main/java/dk/cngroup/lentils/controller/CypherController.java

```diff
@@ -55,8 +64,7 @@ public String saveNewCypher(@Valid @ModelAttribute("command") final CypherFormOb
         }
 
         final Cypher cypher = new Cypher();
-        CypherFormObject.toEntity(cypher, command);
-
+        cypherFormConverter.toEntity(cypher, command);
```

Comment:
Hm, dívám se, že jsem to měl celkem blbě čitelné.
Co u téhle metody prohodit parametry? Aby to bylo jakoby zleva doprava. Command se kopíruje do cypher.

Jsem se na chvíli zasekl, která ta Entita je vlastně která.